common/eth2/src/types.rs

```diff
@@ -954,6 +954,12 @@ pub struct SseHead {
     pub execution_optimistic: bool,
 }
 
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize)]
+
```

Comment:
```suggestion
```

generally good to avoid blank lines between `#[]` and the thing they are applied to